platform/src/main/java/org/stellar/anchor/platform/config/PropertyClientsConfig.java

```diff
@@ -24,14 +24,32 @@ public class PropertyClientsConfig implements ClientsConfig, Validator {
   Map<String, String> domainToClientNameMap = null;
   Map<String, String> signingKeyToClientNameMap = null;
 
+  @PostConstruct
+  public void postConstruct() {
+    clients.forEach(
+        clientConfig -> {
+          if (!isEmpty(clientConfig.getSigningKey())
+              && (clientConfig.getSigningKeys() == null
+                  || clientConfig.getSigningKeys().isEmpty())) {
+            clientConfig.setSigningKeys(setOf(clientConfig.getSigningKey()));
+          }
+          if (!isEmpty(clientConfig.getDomain())
+              && (clientConfig.getDomains() == null || clientConfig.getDomains().isEmpty())) {
+            clientConfig.setDomains(setOf(clientConfig.getDomain()));
+          }
+        });
+  }
+
   @Override
   public ClientsConfig.ClientConfig getClientConfigBySigningKey(String signingKey) {
     if (signingKeyToClientNameMap == null) {
       signingKeyToClientNameMap = Maps.newHashMap();
       clients.forEach(
           clientConfig -> {
-            if (clientConfig.getSigningKey() != null) {
-              signingKeyToClientNameMap.put(clientConfig.getSigningKey(), clientConfig.getName());
+            if (clientConfig.getSigningKeys() != null && !clientConfig.getSigningKeys().isEmpty()) {
```

Comment:
I was hesitate to do that since previously we allow non-custodial client to specify signing key and custodial client to have domain, we might want to keep this until complete deprecation of `domain` and `signing key`? Although it shouldn't have any difference functional wise

Reply:
Let's do that